resources/fonts/sprites.json

```diff
@@ -1,1305 +1,4775 @@
+        "path": "latin/U99-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U100-small.png",
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U101-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U102-small.png",
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U103-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U104-small.png",
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U105-small.png",
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/j-small.png",
+        "x": -1,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U107-small.png",
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/l-small.png",
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U109-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U110-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U111-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U112-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U113-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U114-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U115-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U116-small.png",
+        "y": 1,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U117-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U118-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U119-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U120-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U121-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "latin/U122-small.png",
+        "y": 2,
+        "palette": "keep",
+        "format": "raw"
+    },
+    {
+        "path": "../g2/empty.png",
+        "x": 19,
+        "y": 11
+    },
```

Comment:
Just for my understanding, what is the purpose of these empty images? Padding for certain unprintable characters?

Reply:
Indeed. There are a bunch of control codes between the ASCII base and the first accented characters. It saves a considerable amount of lookup.